internal/crypto/src/tests/fixtures/raw_signature/ps256.pub

```diff
@@ -1,36 +1,76 @@
 -----BEGIN CERTIFICATE-----
```

Comment:
These test fixtures were regenerated to match what was in the `c2pa-rs` test fixture directory.

In a prior PR, I had generated new `ps***` certs and private keys, but that turned out to be unnecessary.